experiment/simulation/index.html

```diff
@@ -1,13 +1,135 @@
-<!DOCTYPE html>
 <html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
-</head>
-<body>
-		<!-- Your code goes here-->
+  <head>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css"
```

Comment:
If we decide to use the beta distribution of a library, we may need to update it later. Please do not use any beta versions except when they provide significant value.